lib/rspec/matchers/built_in/has.rb

```diff
@@ -2,9 +2,11 @@ module RSpec
   module Matchers
     module BuiltIn
       # @api private
```

Comment:
Since this is now home for both dynamic`be_` and `has_`, maybe name the file somehow specifically?

Reply:
I'm thinking it would be good to rename `has.rb` to `dynamic_predicate.rb`. IMO the 3 classes benefit from being in the same file. To help git follow, I'd rename the file after everything else is merged (or I could do it as the very first commit).

Reply:
Makes sense.
Even though probably Github won't probably be able to follow anyway, but it's worth giving it a try.